### PR TITLE
Removed FeaturesReport

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to https://github.com/Particular/NServiceBus/pull/3335

### CODE REVIEW

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -487,4 +487,8 @@ To re-add this header to outgoing messages a [mutator](/nservicebus/pipeline/mes
 
 snippet: 5to6ReAddWinIdNameHeader
 
-Another option is to use your own custom header as illustrated in [Appending username using headers](/samples/username-header/) sample.
\ No newline at end of file
+Another option is to use your own custom header as illustrated in [Appending username using headers](/samples/username-header/) sample.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove "your"

File: nservicebus/upgrades/5to6.md [hunk 2]
```diff
@@ -487,4 +487,8 @@ To re-add this header to outgoing messages a [mutator](/nservicebus/pipeline/mes
...
+
+## Removed FeaturesReport
+
+`FeaturesReport` exposed reporting information about features of a running endpoint instance. It has been internalized. Similarly to previous versions the information is still available if you run the endpoint with log level `DEBUG` and inspect the `DisplayDiagnosticsForFeatures` logger.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove "you"

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: perhaps link to logging?

**Comment by janovesk** [comment]
Updated text at suggested. Merged together with code change in https://github.com/Particular/NServiceBus/pull/3335.

**Pull request merged**